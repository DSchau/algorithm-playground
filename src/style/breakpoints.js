// @flow
const breakpointFn = (breakpoint: number) => {
  return (content: string) => `
@media only screen and (min-width: ${breakpoint}px) {
  ${content}
}
  `;
}

export const SMALL = breakpointFn(0);
export const MEDIUM = breakpointFn(400);
export const LARGE = breakpointFn(768);
