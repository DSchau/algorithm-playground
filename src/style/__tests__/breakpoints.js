import * as BREAKPOINTS from '../breakpoints';

test('it has expected breakpoints', () => {
  expect(Object.keys(BREAKPOINTS)).toMatchSnapshot();
});
