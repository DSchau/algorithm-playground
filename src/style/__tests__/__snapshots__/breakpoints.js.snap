// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it has expected breakpoints 1`] = `
Array [
  "SMALL",
  "MEDIUM",
  "LARGE",
]
`;
